packages/documentation/src/layouts/module-components.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import { graphql } from 'gatsby';
-import MDXRenderer from 'gatsby-mdx/mdx-renderer';
-import { MDXProvider } from '@mdx-js/tag';
+import { MDXProvider } from '@mdx-js/react';
```

Comment:
These changes are necessary due to deprecations/version changes, correct?

Reply:
That's correct, the `gatsby-mdx` plugin is deprecated, and the `@mdx-js/react` dependency is now used to render react.